README.md

```diff
@@ -120,6 +120,28 @@ public class Lookup {
 }
 ```
 
+You can also use the reader object to iterate over the database. 
+The `reader.networks()` and `reader.networksWithin()` methods can 
+be used for this purpose.
+
+```java
+Reader reader = new Reader(file);
+Networks networks = reader.networks(false, Map.class);
+
+while(networks.hasNext()) {
```

Comment:
It would have been nice if we could implement `Iterable`, but I guess that isn't really possible given that we need to pass the class through. I guess it doesn't matter too much.